crates/turborepo-globwatch/Cargo.toml

```diff
@@ -13,8 +13,8 @@ camino = "1.1.4"
 futures = { version = "0.3.26" }
 itertools.workspace = true
 merge-streams = "0.1.2"
-notify = "5.1"
```

Comment:
This is 6.1.1 now

Reply:
the only breaking change is "files and directories moved into a watch folder on Linux will now be reported as rename to events instead of create events" which doesn't affect `turborepo-globwatch`